agents/ocf/HealthCPU.in

```diff
@@ -121,6 +120,10 @@ healthcpu_monitor() {
     if [ -f "${OCF_RESKEY_state}" ]; then
 
         IDLE=$(top -b -n2 | grep Cpu | tail -1 | awk -F",|.[0-9][ %]id" '{ print $4 }')
```

Comment:
Existing code... I don't understand why we run for two iterations. But whatever.